chronos_voldy_cfg/config/.temp/node.id

```diff
@@ -0,0 +1 @@
+0
```

Comment:
@ecarlson156 what are all the `.temp` files. You have a bajillion in there. If it's unintentional, use `.gitignore` 😃